rules/S6304/metadata.json

```
@@ -0,0 +1,40 @@
+{
+  "title": "Having policies granting access to all resources of an AWS account is security-sensitive",
```

S6302 don't have AWS in its title. I think its a good idea as this rule are AWS specific.
What about attaching AWS adjective to "policies"?
```suggestion
  "title": "Having AWS policies granting access to all resources of an account is security-sensitive",
```

---

Seems a good idea